libsolidity/ast/Types.cpp

```diff
@@ -375,6 +375,7 @@ TypePointer Type::commonType(Type const* _a, Type const* _b)
 
 MemberList const& Type::members(ASTNode const* _currentScope) const
 {
+	// TODO _currentScope can be nullptr!
```

Comment:
Will this be fixed in this PR?

Reply:
This was just a note from the beginning of the implementation work. There is an assert below that takes it being nullptr into account, so I assume it is fine.